src/commands/createFunctionApp.ts

```diff
@@ -4,16 +4,23 @@
  *--------------------------------------------------------------------------------------------*/
 
 import { AzureTreeDataProvider, IActionContext, IAzureNode, IAzureParentNode } from 'vscode-azureextensionui';
+import { FunctionAppTreeItem } from '../tree/FunctionAppTreeItem';
 import { nodeUtils } from '../utils/nodeUtils';
 
-export async function createFunctionApp(actionContext: IActionContext, tree: AzureTreeDataProvider, arg?: IAzureParentNode | string): Promise<string> {
+export async function createFunctionApp(actionContext: IActionContext, tree: AzureTreeDataProvider, subscription?: IAzureParentNode | string, resourceGroup?: IAzureParentNode<FunctionAppTreeItem> | string): Promise<string> {
     let node: IAzureParentNode;
-    if (typeof arg === 'string') {
-        node = await nodeUtils.getSubscriptionNode(tree, arg);
-    } else if (!arg) {
+    if (typeof subscription === 'string') {
+        node = await nodeUtils.getSubscriptionNode(tree, subscription);
+    } else if (!subscription) {
         node = <IAzureParentNode>await tree.showNodePicker(AzureTreeDataProvider.subscriptionContextValue);
     } else {
-        node = arg;
+        node = subscription;
+    }
+    if (resourceGroup) {
+        if (typeof resourceGroup !== 'string') {
+            resourceGroup = resourceGroup.treeItem.client.resourceGroup;
+        }
+        actionContext.properties.resourceGroup = resourceGroup;
```

Comment:
`actionContext.properties` is used for telemetry and we don't want to collect this in telemetry

We'll have to find a different way to pass the resource group along

Reply:
There are UserSettings here:
https://github.com/Microsoft/vscode-azurefunctions/blob/56a097178f1deac96f69cac174be26f2c2899825/src/commands/createFunction/createFunction.ts#L150

But it seems like it's used for appSettings which get passed into the ActionContext.  Should we just create a new property?